src/Electron/electron.ts

```diff
@@ -121,7 +122,7 @@ export class ElectronMessageBus implements MessageBus {
         }
 
         // Broadcast to all windows or to the individual targeted window
-        if (this.browserWindow) {
```

Comment:
Added some conditional checks for safety that were more for unit test mocking then production use but good to have regardless.